outofcore/include/pcl/outofcore/octree_base.h

```diff
@@ -620,6 +619,11 @@ namespace pcl
         void
         flushToDiskLazy ();
 
+        /** \brief Flush empty nodes only */
+        PCL_DEPRECATED(1, 13, "Use DeAllocEmptyNodeCache() with no arguments instead")
+        void
+        DeAllocEmptyNodeCache ();
```

Comment:
Needs a definition
```suggestion
        DeAllocEmptyNodeCache () {
          DeAllocEmptyNodeCache(root_node_);
        }
```
```

Reply:
Actually it's probably best left exactly as it was in the hpp